apps/site-landing/src/containers/ReturnProfile/data/article.test.ts

```diff
@@ -38,12 +38,12 @@ it ('Matches article with ShockAbsorber', () => {
       // We protect up to $500
       maximumProtected: new Decimal(500),
       // for a max drop of 50%
-      absorbed: new Decimal(0.50),
+      cushionDown: new Decimal(0.50),
```

Comment:
I couldn't remember what up/down, renamed because both up/down operations have the same intended effect (to smooth up the jumps/valleys)